pages/homepage-tools.js

```diff
@@ -0,0 +1,116 @@
+import Link from 'next/link'
+
+import theme from '@/styles/theme'
+
+import BanSearch from '@/components/ban-search'
+import {Database, FileText, MapPin} from 'react-feather'
```

Comment:
Attention à l'ordre des imports et le groupement afin de rester homogène avec le reste du code existant.

```suggestion
import Link from 'next/link'
import {Database, FileText, MapPin} from 'react-feather'

import theme from '@/styles/theme'

import BanSearch from '@/components/ban-search'
```